config/routes.rb

```diff
@@ -14,7 +14,7 @@
   get "/auth/dfe/signout", to: "sessions#destroy"
   get "/auth/failure", to: "sessions#failure"
 
-  root to: "providers#index"
+  root to: "pages#start"
```

Comment:
Should this be optional?

Reply:
i.e. can we switch this based on `Settings.show_start_page`, so we can turn it off without redeploying.

Reply:
MVP

Reply:
:shipit:

Reply:
Happy to approve once you're happy with it to come out of draft.